src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -744,11 +746,12 @@ def filter_and_paginate(
                 [
                     Session,
                     Union[Select[AnySchema], SelectOfScalar[AnySchema]],
-                    BaseFilterModel,
+                    BaseFilter,
                 ],
                 List[AnySchema],
             ]
         ] = None,
+        hydrate: Optional[bool] = None,
```

Comment:
Why is this optional?
```suggestion
        hydrate: bool = False,
```

Reply:
Good catch! Fixed it.